scripts/build/options.js

```diff
@@ -4,12 +4,11 @@ const os = require("os");
 
 /** @type {CommandLineOptions} */
 module.exports = minimist(process.argv.slice(2), {
-    boolean: ["debug", "dirty", "light", "colors", "lint", "lkg", "soft", "fix", "failed", "keepFailed", "force", "built"],
-    string: ["browser", "tests", "inspect", "host", "reporter", "stackTraceLimit", "timeout", "shards", "shardId"],
+    boolean: ["dirty", "light", "colors", "lint", "lkg", "soft", "fix", "failed", "keepFailed", "force", "built"],
+    string: ["browser", "tests", "break", "host", "reporter", "stackTraceLimit", "timeout", "shards", "shardId"],
     alias: {
         "b": "browser",
-        "d": ["debug", "debug-brk"],
-        "i": ["inspect", "inspect-brk"],
+        "i": ["inspect", "inspect-brk", "break"],

```

Comment:
should we also add `debug` here, on the offchance it both works and someone's muscle memory causes them to pass `--debug`?

Reply:
```suggestion
        "i": ["inspect", "inspect-brk", "break", "debug", "debug-brk"],
```